content/tutorials/polygon-bridge.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Polygon (ex Matic)
+description: 
+---
+
+## Intro to Polygon's Bridge
+
+The Polygon Network (ex Matic) provide us with a bridge (connecting Ethereum & Polygon blockchains), and a dedicated [wallet](https://wallet.matic.network) that simplify the steps of transferring digital assets between the two networks. The wallet connects to your account via Metamask (or any of the other supported wallets). 
+
```

Comment:
this could use a bulleted lists with the main links mentioned throughout this tutorial for enhanced scannability, like:

- [Matic Wallet →](https://wallet.matic.network)
- [OCEAN-Matic Bridge →](#)

Reply:
Agree, I'll check how I can update this paragraph.